Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import telebot\r\n\r\ntoken = \"7875328646:AAEnQegv0NAX3qnEs01m-Qbxznnn7zijH8k\"\r\nbot = telebot.TeleBot(token)\r\n\r\nbot.polling()
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 6cc4fc2ed67b0b41efb57aff7fcdefef0f5a7d15)
+++ b/main.py	(date 1747130349852)
@@ -3,4 +3,4 @@
 token = "7875328646:AAEnQegv0NAX3qnEs01m-Qbxznnn7zijH8k"
 bot = telebot.TeleBot(token)
 
-bot.polling()
\ No newline at end of file
+bot.polling()
Index: .idea/PythonProject.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"PYTHON_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <excludeFolder url=\"file://$MODULE_DIR$/.venv\" />\r\n    </content>\r\n    <orderEntry type=\"jdk\" jdkName=\"Python 3.13 virtualenv at C:\\Users\\kotuk\\PycharmProjects\\historyBot\\.venv\" jdkType=\"Python SDK\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/PythonProject.iml b/.idea/PythonProject.iml
--- a/.idea/PythonProject.iml	(revision 6cc4fc2ed67b0b41efb57aff7fcdefef0f5a7d15)
+++ b/.idea/PythonProject.iml	(date 1747130349870)
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/.venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.13 virtualenv at C:\Users\kotuk\PycharmProjects\historyBot\.venv" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.13 (historyBot)" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
 </module>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"Black\">\r\n    <option name=\"sdkName\" value=\"Python 3.13 (PythonProject)\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.13 virtualenv at C:\\Users\\kotuk\\PycharmProjects\\historyBot\\.venv\" project-jdk-type=\"Python SDK\" />\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 6cc4fc2ed67b0b41efb57aff7fcdefef0f5a7d15)
+++ b/.idea/misc.xml	(date 1747130349877)
@@ -3,5 +3,5 @@
   <component name="Black">
     <option name="sdkName" value="Python 3.13 (PythonProject)" />
   </component>
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.13 virtualenv at C:\Users\kotuk\PycharmProjects\historyBot\.venv" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.13 (historyBot)" project-jdk-type="Python SDK" />
 </project>
\ No newline at end of file
